packages/trader/build/loaders-config.js

```diff
@@ -2,9 +2,6 @@ const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const path = require('path');
 
 const js_loaders = [
-    {
-        loader: '@deriv/shared/src/loaders/react-import-loader.js',
```

Comment:
Why do we need this change?

Reply:
It is removed from some packages and we have to remove it from all the packages where it is still present.